resources/src/main/java/org/robolectric/res/android/Util.java

```diff
@@ -52,33 +54,39 @@ public static boolean isTruthy(Object o) {
     return o != null;
   }
 
-  static void ALOGD(String message, Object... args) {
+  @FormatMethod
+  static void ALOGD(@FormatString String message, Object... args) {
     if (DEBUG) {
       System.out.println("DEBUG: " + String.format(message, args));
     }
   }
 
-  static void ALOGW(String message, Object... args) {
+  @FormatMethod
```

Comment:
Looks like only `FormatMethod` is needed to fix `AnnotateFormatMethod` warnings. Could we reset above ALOGI/D/V changes?

Reply:
But `ALOGI/D/V` methods already have `@FormatMethod` annotation

- ALOGI: https://github.com/robolectric/robolectric/pull/7353/files/2b875a8c788b9f99401768154b97f28907622208#diff-48bfa6ff48c8f002369075c214ef80a1cf8435ee90bc86d7f63a1dd012ab4f01R76-R77
- ALOGV: https://github.com/robolectric/robolectric/pull/7353/files/2b875a8c788b9f99401768154b97f28907622208#diff-48bfa6ff48c8f002369075c214ef80a1cf8435ee90bc86d7f63a1dd012ab4f01R69-R70
- ALOGD: https://github.com/robolectric/robolectric/pull/7353/files/2b875a8c788b9f99401768154b97f28907622208#diff-48bfa6ff48c8f002369075c214ef80a1cf8435ee90bc86d7f63a1dd012ab4f01R57-R58

Reply:
Could we only mark ALOGI/D/V method with `FormatMethod` and don't change ALOGI/D/V usage?

Reply:
removed the changes from the calling place of `ALOGI/D/V` usages